vagrant/k8s/setup_cluster.sh

```diff
@@ -60,39 +60,61 @@ function GetContrib {
 	popd
 }
 
-# kubernetes version to use -- defaults to v1.2.3
-: ${k8sVer:=v1.2.3}
+# kubernetes version to use -- legacy is v1.2.3
+#: ${k8sVer:=v1.2.3}
+
+# kubectl installation mechanism default is v1.4.4
+: ${k8sVer:=v1.4.4}
```

Comment:
Have we verified that demos described in https://github.com/contiv/netplugin/tree/master/mgmtfn/k8splugin work as before?